react-app/src/components/Extract.js

```diff
@@ -8,12 +9,18 @@ class Extract extends React.Component {
   constructor(props) {
     super(props);
 
+    const defaultPathToConfig = path.join('test', 'config', 'csv.config.json');
```

Comment:
I made a comment about this above in the main comment box, but if we do rename the folder that has the config and sampel-client-data, we will need to update this path here.